Library/Formula/freeimage.rb

```diff
@@ -26,6 +26,13 @@ class Freeimage < Formula
 
   patch :DATA
 
+  # fix GCC 5.0
+  patch do
+    url "https://raw.githubusercontent.com/Homebrew/patches/master/freeimage/gcc5.diff"
```

Comment:
Can you use a commit reference here in case we ever delete the patch from master or something? Thanks.